layers/+completion/helm/funcs.el

```diff
@@ -513,6 +513,23 @@ Removes the automatic guessing of the initial value based on thing at point. "
     (set-text-properties 0 (length input) nil input)
     (helm-find-files-1 input)))
 
+;; ido-mode remaps some commands to ido counterparts.  We want default Emacs key
+;; bindings (those under C-x) to use ido, but we want to use the original
+;; commands in Spacemacs key bindings (those under M-m or SPC) so that they use
+;; `read-file-name-function', `completing-read-function',
+;; `completion-in-region-function', etc. configured by Helm or Ivy etc.  The
+;; following aliases allow us to bind Spacemacs keys to the original commands.
+
+(defalias 'spacemacs/find-file-other-frame 'find-find-other-frame)
```

Comment:
`find-find-other-frame` --> `find-file-other-frame`

Reply:
Sorry! That's embarrassing—and weird because I thought I tested every one of these bindings...